tests/test_tuya_smoke.py

```diff
@@ -66,3 +70,59 @@ async def test_tuya_smoke_sensor_attribute_update(zigpy_device_from_quirk, quirk
     assert len(ias_listener.attribute_updates) == 2
     assert ias_listener.attribute_updates[1][0] == zone_status_id
     assert ias_listener.attribute_updates[1][1] == IasZone.ZoneStatus.Alarm_1
+
+
+@pytest.mark.parametrize(
+    "model,manuf",
+    [
+        ("_TZE200_dq1mfjug", "TS0601"),
+        ("_TZE200_m9skfctm", "TS0601"),
+        ("_TZE200_ntcy3xu1", "TS0601"),
+        ("_TZE200_rccxox8p", "TS0601"),
+        ("_TZE200_vzekyi4c", "TS0601"),
+        ("_TZE204_vawy74yh", "TS0601"),
+        ("_TZE204_ntcy3xu1", "TS0601"),
+        ("_TZE284_0zaf1cr8", "TS0601"),
+    ],
+)
+async def test_handle_get_data(zigpy_device_from_v2_quirk, model, manuf):
+    """Test handle_get_data for multiple attributes."""
+
+    zone_status_id = IasZone.AttributeDefs.zone_status.id
+
+    quirked = zigpy_device_from_v2_quirk(model, manuf)
+    ep = quirked.endpoints[1]
+
+    assert ep.basic is not None
+    assert isinstance(ep.basic, Basic)
+
+    assert ep.tuya_manufacturer is not None
+    assert isinstance(ep.tuya_manufacturer, TuyaMCUCluster)
+
+    message = b"\x09\x39\x02\x00\x11\x01\x04\x00\x01\x00"
+    hdr, data = ep.tuya_manufacturer.deserialize(message)
+
+    status = ep.tuya_manufacturer.handle_get_data(data.data)
+    assert status == foundation.Status.SUCCESS
+
+    assert ep.ias_zone.get(zone_status_id) == IasZone.ZoneStatus.Alarm_1
```

Comment:
Most of the time, we use a `ClusterListener` to check for attribute updates (like the test above this one does) (where `opple_cluster` should be renamed to `tuya_cluster) instead of checking attribute cache.
This should be fine as well, but maybe we wanna make it consistent and use `ClusterListener` here too.

Reply:
Updated to use `ClusterListener`, retained using the actual message captured from the device though, let me know your thoughts.